src/views/apps/AppFeedImporter.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <Application>
+    <template #settings-append>
+      <div id="collapse-settings" class="collapse show">
+        <div>
+          <form autocomplete="off">
+            <div role="tablist" class="card border mb-2">
+              <div id="collapse-0" class="card-body collapse show">
+                <div>
+                  <div>
+                    <div class="form-group">
+                      <h3>Importação de planílha (CSV/XLSX)</h3>
+                      <span
+                        >Importação baseada no seguinte
+                        <a
+                          href="https://firebasestorage.googleapis.com/v0/b/ecom-feed-importer.appspot.com/o/public%2Ftemplate.csv?alt=media&token=85526ea3-ce75-4724-8184-cbae766e78fa"
+                        >
+                          exemplo.
+                        </a>
+                      </span>
+                      <b-form-file
+                        v-model="file"
+                        placeholder="Escolha o arquivo (csv/xlsx)..."
+                        drop-placeholder="Drop file here..."
```

Comment:
```suggestion
                        placeholder="Escolha o arquivo (CVS ou XLSX)..."
                        drop-placeholder="Arraste o arquivo aqui..."
```

O ideal é usar github.com/ecomplus/i18n , se for mockado tem que ser em pt né..